spec/unit/reports/servicenow/event_spec.rb

```diff
@@ -50,6 +51,19 @@
     processor.process
   end
 
+  it 'includes multiple labels in a description' do
```

Comment:
Let's codify the "audit" status here to either be explicitly included or excluded from the list of labels since the logic around audit is a bit different.